projects/cashmere/src/lib/chip/chip.component.ts

```diff
@@ -1,18 +1,21 @@
 import {Component, Input, ViewEncapsulation} from '@angular/core';
 import {parseBooleanAttribute} from '../util';
 
+/** Chips represent complex entities in small blocks, such as filters, contacts, or system information */
 @Component({
     selector: 'hc-chip',
     template: `<div [class]="color" [ngClass]="{'close': _action, 'chip': true}"><ng-content></ng-content></div>`,
     styleUrls: ['./chip.component.scss'],
     encapsulation: ViewEncapsulation.None
 })
 export class ChipComponent {
-    _action: boolean = false;
+    private _action: boolean = false;
```

Comment:
Private needs to be left off since it's used in the template

Reply:
or _action in the template needs to be changed to use the action property

Reply:
Good catch @jtrujill - and thanks for talking this through with me.  Completely missed this before, so the docs update is making our code better!  I adjusted the templates in chips and chip-row, checked a build and everything still works fine.